cmd/zed/dev/command.go

```diff
@@ -0,0 +1,26 @@
+package dev
+
+import (
+	"flag"
+
+	"github.com/brimdata/zed/cmd/zed/root"
+	"github.com/brimdata/zed/pkg/charm"
+)
+
+var Cmd = &charm.Spec{
+	Name:  "dev",
+	Usage: "dev [options] sub-command",
```

Comment:
`dev` doesn't have any options, but the dev commands do take arguments.
```suggestion
	Usage: "dev sub-command [arguments...]",
```

Reply:
Whoops!